tasks/main-clevis.yml

```diff
@@ -4,6 +4,14 @@
     name: "{{ __nbde_client_packages }}"
     state: present
 
+- name: Update clevis dracut config to enable networking if Tang binding is found
+  lineinfile:
+    path: /etc/dracut.conf.d/clevis.conf
```

Comment:
Perhaps name the file `nbde_client.conf`, to make it more obvious it was added by this role?